### DISCUSSION TITLE
When parse object with an empty array it return []

### DISCUSSION THREAD

**Discussion by namantam1** [eemeli/yaml#482] [Q&A] [answered]
Is this expected behavior?

```js
// when
stringify({arr: [1, 2]})
// returns: 'arr:\n  - 1\n  - 2\n'


// when
stringify({arr: []})
// returns: 'arr: []\n'
```

I was expecting blank array to be `arr: \n`.

**Also, how can I omit blank array fields?**

**Comment by eemeli** [comment, marked as answer]
`[]` is the only available YAML representation of an empty array. The empty value you're expecting represents a null value.

To get what you want, I'd recommend using a replacer to replace empty arrays with `undefined`, which is ignored: https://eemeli.org/yaml/#yaml-stringify
